docs/product/explore/session-replay/web/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Session Replay for Web"
+sidebar_order: 10
+description: "Use Session Replay to get video-like reproductions of user interactions to improve your website or app experience."
```

Comment:
```suggestion
description: "Use Session Replay to get video-like reproductions of user interactions to improve your website experience."
```
since this one is web specific